library/OrderCloud.Catalyst/DataMovement/ListAllAsync/README.md

```diff
@@ -0,0 +1,33 @@
+﻿# ListAllAsync Function
+
+## Purpose
+
+The List function provides a way to retirieve all elements of a specific type from OrderCloud. Since data is returned paginated in OrderCloud, this function allows calling out for multiple pages of data without any more implementation or looping logic.
+
+## Parameters
+
+The List and ListWithFacets function require the same parameter.
+
+-listFunc: This parameter is the function you intend to get data from. Within your function call, you are able to set a few different parameters to decide how many items you want to retrieve. Those parameters are listed below. Also noting these properties are deptermined by the OrderCloud SDK depending on the resource.
+
+- page: page sigifies what page is currently being retrieved. This will dynamically update as the function continues to loop through.
+- pageSize: this determines how many records you want to return with each page.
+- filter: optional parameter to filter results based on a specific property.
```

Comment:
Can we link to the filtering docs? https://ordercloud.io/features/advanced-querying#filtering